sdk/state/ingest.go

```diff
@@ -28,7 +31,13 @@ func (c *Channel) IngestTx(txXDR, resultXDR, resultMetaXDR string) error {
 		return fmt.Errorf("transaction unrecognized")
 	}
 
-	c.ingestTxToUpdateInitiatorEscrowAccountSequence(tx)
+	err = c.ingestTxToUpdateInitiatorEscrowAccountSequence(tx)
+	if errors.Is(err, errOldSeqNum) {
+		return nil
+	}
+	if err != nil {
+		return err
+	}
```

Comment:
❗We should keep processing the tx if it's an old seq num because we might want to ingest it as an unauthorized tx or some other future reason. Inside the ingest tx that updates seq num we should have it exit early if it is a lower number. 

So I think we need this change here:
```suggestion
	if err != nil {
		return err
	}
```

And another change below.

Reply:
sgtm 
https://github.com/stellar/experimental-payment-channels/pull/312/commits/642ef2c69a783310938bc1ded660feaa72853c67